app/prisma/schema.prisma

```diff
@@ -189,15 +188,15 @@ enum DatasetFileUploadStatus {
 model DatasetFileUpload {
     id String @id @default(uuid()) @db.Uuid
 
-    datasetId String   @db.Uuid
-    dataset   Dataset @relation(fields: [datasetId], references: [id], onDelete: Cascade)
-    blobName  String
-    fileName String
-    fileSize Int
-    progress Int @default(0) // Percentage
-    status   DatasetFileUploadStatus @default(PENDING)
-    uploadedAt DateTime
-    visible  Boolean @default(true)
+    datasetId    String                  @db.Uuid
```

Comment:
Do you have a formatter for prisma files?

Reply:
Yeah just the one built into their vs code [extension](https://marketplace.visualstudio.com/items?itemName=Prisma.prisma)